js/msp.js

```diff
@@ -1001,15 +1003,16 @@ var MSP = {
                 }
                 break;
             case MSP_codes.MSP_SPECIAL_PARAMETERS:
```

Comment:
I thought with 2.9.1 we'd be able to get rid of `MSP_SPECIAL_PARAMETERS` altogether?